components/docs-chef-io/content/automate/servicenow_integration.md

```diff
@@ -0,0 +1,257 @@
++++
+title = "ServiceNow Integration"
+
+draft = false
+
+gh_repo = "automate"
+
+[menu]
+  [menu.automate]
+    title = "ServiceNow Integration"
```

Comment:
We have two items in the menu called ServiceNow Integration. Can we make them (integration app and incident app) more differentiable?